src/endpoints/user/addUserHobby.ts

```diff
@@ -0,0 +1,21 @@
+import {IncomingMessage, ServerResponse} from 'http'
+import { UserService } from '../../userService';
+import { getUserIdFromRequest, parsBodyRequest } from '../../utils';
+import { PATHS } from '../../constants';
+
+export function addUserHobbyRoute(req: IncomingMessage, res: ServerResponse) {
+    const userService = UserService.getInstance();
+    const userId: number = getUserIdFromRequest(req, res, PATHS.userHobby);
+
+    parsBodyRequest(req, res, (hobby: string) => {      
+        const user = userService.addHobbyToUser(userId, hobby);
+        if (user && Object.keys(user).length) {
+            res.statusCode = 200;
+            res.end(JSON.stringify(user));
+            return;
+        } else {
+            res.statusCode = 204;
```

Comment:
I think in this case user is not found, so we should return status code 404 and message: User not found